### DISCUSSION TITLE
Submission of application rejected

### DISCUSSION THREAD

**Discussion by zhangyuguai** [community/community#143851] [GitHub Education]
### Hi everyone,

![Snipaste_2024-11-07_17-58-30](https://github.com/user-attachments/assets/f806d028-3d5e-4397-b2c6-527ca4496f5d)
After submitting an application and getting rejected, I am not able to continue submitting applications, why is that? Can anyone tell me how to fix it?

**Comment by mastertanzeel** [comment]
When you get rejected for the GitHub Student Developer Pack, GitHub generally places a limitation on immediately resubmitting an application to prevent repeated submissions without addressing the issues. Here’s what you can do:

1. Check GitHub’s Reapplication Policy:
   - GitHub sometimes has a cooldown period after a rejection. You may need to wait a certain period (such as a week or a month) before submitting a new application. This is to allow time for you to make the necessary updates and prevent repetitive rejections.

2. Contact GitHub Support:
   - Since you’re unable to reapply, you can contact GitHub Support directly for assistance. Explain that you were rejected, have made the recommended updates, and would like to reapply.
   - To do this:
     - Go to [GitHub Support](https://support.github.com/contact) and select "GitHub Education" in the support request options.
     - Describe your situation, detailing the changes you’ve made to meet the requirements.

3. Review and Follow the Rejection Reasons Carefully:
   - GitHub’s rejection reasons are usually very specific. Make sure you’ve followed all the recommended actions exactly as outlined.
   - Double-check each step (like 2FA, profile information, and billing info) and ensure you’ve logged out and back in to sync changes with GitHub.

4. Check for Account Status Updates:
   - Sometimes GitHub might process your updates and reconsider your status without requiring a new application. After you’ve made the necessary changes, check your email or GitHub notifications periodically to see if GitHub automatically updates your application status.

5. Alternative Proof (if applicable):
   - If GitHub’s response indicates issues with verifying your student status, see if there are alternative documents you can provide, like an enrollment verification letter from your school or other proof of student status.

These steps should help you resolve the issue and allow you to apply again once the reapplication option becomes available.
